userspace/files/avahi-override.conf

```diff
@@ -0,0 +1,3 @@
+[Unit]
+After=network.target
+Wants=network.target
```

Comment:
why do you need the override?

Reply:
override is needed because "set-hostname.sh" is called by the serial-hostname.service.  That service has the requirement "Before=network.target".  I could put "After=serial-hostname.service" but i didn't really want to create a dependency on that service existing.  Without this code, avahi starts when the hostname is still "comma" and not "comma-serial".  Let me know what you want to do here, if you want me to leave a comment in the code or make a change.